iOS_SDK/OneSignalSDK/Source/OSInAppMessage.m

```diff
@@ -195,13 +195,15 @@ - (void)encodeWithCoder:(NSCoder *)encoder {
     [encoder encodeObject:_messageId forKey:@"messageId"];
     [encoder encodeObject:_variants forKey:@"variants"];
     [encoder encodeObject:_triggers forKey:@"triggers"];
+    [encoder encodeObject:_displayStats forKey:@"redisplay"];
```

Comment:
Use the variable name as the key
`[encoder encodeObject:_displayStats forKey:@"displayStats"];`